pyt/__main__.py

```diff
@@ -30,7 +30,25 @@
 )
 
 
-def main(command_line_args=sys.argv[1:]):  # noqa: C901
+def discover_files(targets, excluded_files, recursive=False):
+    included_files = list()
+    excluded_list = excluded_files.split(",")
+
+    for target in targets:
+        if os.path.isdir(target):
+            if recursive:
+                for root, dirs, files in os.walk(target):
+                    for f in files:
+                        fullpath = os.path.join(root, f)
+                        if os.path.splitext(fullpath)[1] == '.py' and fullpath.split("/")[-1] not in excluded_list:
+                            included_files.append(fullpath)
+        else:
+            if target not in excluded_list:
+                included_files.append(targets[0])
```

Comment:
So if targets is a list of files, e.g. `python -m pyt examples/vulnerable_code/command_injection.py examples/vulnerable_code/XSS.py`, then `discover_files` will return the first file N times. (Where N is the len of targets.)